src/common/components/repositories-home/index.js

```diff
@@ -119,10 +119,19 @@ class RepositoriesHome extends Component {
       );
     }
 
+    // switch over to snapcraft.io/<snap_name>/builds notification
+    const infoNotification = (
+      <Notification appearance='information'>
+        We are updating the way you build from GitHub to integrate better within
+        Snapcraft. <Anchor href="#">Learn more</Anchor>
```

Comment:
It seems to be adding a linked button:

<img width="1312" alt="Screenshot 2020-04-01 at 14 47 01" src="https://user-images.githubusercontent.com/83575/78138748-b8ba8700-7427-11ea-863e-d47b19c17e79.png">

For a simple external link just `<a href=...>` should be enough.

Reply:
ok, thanks